app/bundles/EmailBundle/Resources/views/Email/details.html.twig

```diff
@@ -151,7 +151,7 @@
               <div class="collapse" id="email-details">
                   <div class="pr-md pl-md pb-md">
                       <div class="panel shd-none mb-0">
-                          <table class="table table-bordered table-striped mb-0">
```

Comment:
Can you double-check that the `table-hover` class is in all the tables to have a unified experience?

Reply:
I was afraid to add it somewhere inadequate
What can you say about the size of this PR? Is it better to change related things all in one or separate in smaller ones next time?

Reply:
I think that the size is OK with this PR. I saw that you were adding the `table-hover` in [other places](https://github.com/mautic/mautic/pull/13748/files#diff-27df81a99f1c08bbd4b906f297c1480423503598bb618584de1e4e34b82da1c3R3) and it was missing on the one place I was testing this on. So I thought it would be good to have the table consistent.

Reply:
Done!